packages/graphback-datasync/src/deltaHelper.ts

```diff
@@ -0,0 +1,46 @@
+import { ModelDefinition, getTableName } from '@graphback/core';
+import { Db, Collection, ObjectId } from "mongodb";
+import { DataSyncFieldNames } from './util';
+
+export function getDeltaTableName(tableName: string) {
+  return `${tableName}Delta`
+}
+
+/**
+ * M
+ */
+export class MongoDeltaHelper {
+  protected db: Db;
+  protected collectionName: string;
+  public constructor(model: ModelDefinition, db: Db) {
+    this.db = db;
+    this.collectionName = getDeltaTableName(getTableName(model.graphqlType));
+  }
+
+  public async insertDiff(updatedDocument: any) {
+    const version = updatedDocument[DataSyncFieldNames.version];
+    const { _id } = updatedDocument;
```

Comment:
I believe this could be a one-liner:

```suggestion
const { _id, [DataSyncFieldNames.version]: version } = updatedDocument;
```